hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/impl/ZookeeperFederationStateStore.java

```diff
@@ -104,29 +107,39 @@
 public class ZookeeperFederationStateStore implements FederationStateStore {
 
   private static final Logger LOG =
-      LoggerFactory.getLogger(ZookeeperFederationStateStore.class);
+          LoggerFactory.getLogger(ZookeeperFederationStateStore.class);
```

Comment:
Avoid

Reply:
Thanks for your help reviewing the code, I will fix it.